### PR TITLE
test/language/literals/regexp/*: early error test fixups

### PR DESCRIPTION
**Pull request by rwaldron**

### CODE REVIEW

File: test/language/literals/regexp/7.8.5-1gs.js [hunk 1]
```diff
@@ -2,13 +2,29 @@
...
+    RegularExpressionChars ::
+      [empty]
+      RegularExpressionChars RegularExpressionChar
+
+  RegularExpressionFirstChar ::
+    RegularExpressionNonTerminatorbut not one of * or \ or / or [
+    RegularExpressionBackslashSequence
+    RegularExpressionClass
+
+description: RegularExpressionBody will never be empty, because RegularExpressionFirstChar must always be _something_. // is a comment.
```
Comment (leobalter) [inline test/language/literals/regexp/7.8.5-1gs.js]: +1

File: test/language/literals/regexp/S7.8.5_A1.1_T1.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ description: Without eval
 //CHECK#1
 if (/1/.source !== "1") {
   $ERROR('#1: /1/');
-}   
+}
```
Comment (leobalter) [inline test/language/literals/regexp/S7.8.5_A1.1_T1.js]: are you sure this is not checking for a 2 spaces chars syntax?

Comment (rwaldron) [reply test/language/literals/regexp/S7.8.5_A1.1_T1.js]: R-

File: test/language/literals/regexp/S7.8.5_A1.2_T1.js [hunk 3]
```diff
@@ -2,15 +2,21 @@
 // This code is governed by the BSD license found in the LICENSE file.
 
 /*---
-info: "RegularExpressionFirstChar :: * or \\ or / or [empty] is incorrect"
-es5id: 7.8.5_A1.2_T1
-description: "*"
```
Comment (leobalter) [inline test/language/literals/regexp/S7.8.5_A1.2_T1.js]: *, just testing everything, lol

File: test/language/literals/regexp/S7.8.5_A1.2_T2.js [hunk 4]
```diff
@@ -2,15 +2,27 @@
...
+    [empty]
+    RegularExpressionChars RegularExpressionChar
+
+  RegularExpressionFirstChar ::
+    RegularExpressionNonTerminator but not one of * or \ or / or [
+
+  RegularExpressionNonTerminator ::
+    SourceCharacterbut not LineTerminator
+
+description: RegularExpressionFirstChar :: RegularExpressionNonTerminator but not one of * or \ or / or [, \ case
```
Comment (leobalter) [inline test/language/literals/regexp/S7.8.5_A1.2_T2.js]: yaml parse is failing, I guess  you might need to scape this in quotes or using a `description: > \n`  format

File: test/language/literals/regexp/S7.8.5_A1.2_T1.js (hunk 3 above)
Comment (rwaldron) [reply test/language/literals/regexp/S7.8.5_A1.2_T1.js]: 🤣

File: test/language/literals/regexp/S7.8.5_A1.2_T2.js (hunk 4 above)
Comment (rwaldron) [reply test/language/literals/regexp/S7.8.5_A1.2_T2.js]: Whoops!

Comment (rwaldron) [reply test/language/literals/regexp/S7.8.5_A1.2_T2.js]: Fixed

**Pull request merged**